src/commands/apps-sdkconfig.ts

```diff
@@ -0,0 +1,117 @@
+import * as ora from "ora";
+import * as fs from "fs-extra";
+
+import * as Command from "../command";
+import {
+  AppMetadata,
+  AppPlatform,
+  getAppConfig,
+  getAppPlatform,
+  listFirebaseApps,
+} from "../management/apps";
+import * as FirebaseError from "../error";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+import { promptOnce } from "../prompt";
+
+async function selectAppInteractively(
+  options: any,
+  appPlatform: AppPlatform
+): Promise<AppMetadata> {
+  let projectId = options.project;
+  if (!projectId) {
+    projectId = await promptOnce({
+      type: "input",
+      message: "Please input the project ID you would like to use:",
+    });
+  }
+
+  const apps: AppMetadata[] = await listFirebaseApps(projectId, appPlatform);
+  if (apps.length === 0) {
+    throw new FirebaseError("There is no app associated with this Firebase project");
+  }
+
+  const choices = apps.map((app: any) => {
+    return {
+      name:
+        `${app.displayName || app.bundleId || app.packageName}` +
+        ` - ${app.appId} (${app.platform})`,
+      value: app,
+    };
+  });
+
+  return await promptOnce({
+    type: "list",
+    name: "id",
+    message:
+      `Select the ${appPlatform === AppPlatform.ANY ? "" : appPlatform + " "}` +
+      "app to get the configuration data:",
+    choices,
+  });
+}
+
+module.exports = new Command("apps:sdkconfig [platform] [appId]")
+  .description(
+    "print the configuration of a Firebase app. " +
+      "[platform] can be IOS, ANDROID or WEB (case insensitive)"
+  )
+  .option("-o, --out", "(optional) write config output to a file")
```

Comment:
Should this have a parameter? `"-o, --out [file]"`?

Reply:
We agreed in the [propopal](https://docs.google.com/document/d/1jkwfbu-3H4MNmvKXloMl5SonYxodWuFenZaYLF2_tFo/edit#heading=h.9l38lsxtecky) not adding `[file]` parameter here for simplicity. 

Adding @nbegley to confirm.

Reply:
In this case, the intention was that, when the `--out` option is used in the command, the config data will be saved to a file using the recommended filename returned by the getconfig API (currently the platform-dependent values are `GoogleService-Info.plist`, `google-services.json`, or `google-config.js`). If the user wants to save to a different filename/path, they can omit `--out` and pipe/redirect the output themselves.

Alternatively, we could support `--out [filename]` like @bkendall suggests, and still use the Firebase-recommended filename when `--out` is set without a user-provided filename. I think I prefer this option.

Reply:
Added support for `"-o, --out [file]"`